bpfd-operator/bundle/manifests/bpfd.io_tcprograms.yaml

```diff
@@ -0,0 +1,315 @@
+                            merge patch.
+                          items:
+                            type: string
+                          type: array
+                      required:
+                      - key
+                      - operator
+                      type: object
+                    type: array
+                  matchLabels:
+                    additionalProperties:
+                      type: string
+                    description: matchLabels is a map of {key,value} pairs. A single
+                      {key,value} in the matchLabels map is equivalent to an element
+                      of matchExpressions, whose key field is "key", the operator
+                      is "In", and the values array contains only "value". The requirements
+                      are ANDed.
+                    type: object
+                type: object
+                x-kubernetes-map-type: atomic
+              priority:
+                description: Priority specifies the priority of the tc program in
+                  relation to other programs of the same type with the same attach
+                  point. It is a value from 0 to 1000 where lower values have higher
+                  precedence.
+                format: int32
+                maximum: 1000
+                minimum: 0
+                type: integer
+              proceedon:
+                default:
+                - pipe
+                - dispatcher_return
+                description: ProceedOn allows the user to call other tc programs in
+                  chain on this exit code. Multiple values are supported by repeating
+                  the parameter.
+                items:
+                  enum:
+                  - unspec
+                  - ok
+                  - reclassify
+                  - shot
+                  - pipe
+                  - stolen
+                  - queued
+                  - repeat
+                  - redirect
+                  - trap
+                  - dispatcher_return
+                  type: string
+                maxItems: 11
+                type: array
+              sectionname:
+                description: SectionName is the the section name described in the
+                  bpf Program
+                type: string
+            required:
+            - bytecode
+            - direction
+            - interfaceselector
+            - nodeselector
+            - priority
+            - sectionname
+            type: object
+          status:
+            description: TcProgramStatus defines the observed state of TcProgram
+            properties:
+              conditions:
+                description: Conditions houses the global cluster state for the TcProgram.
+                  The explicit condition types are defined internally.
+                items:
+                  description: "Condition contains details for one aspect of the current
```

Comment:
When is this printed pretty?

Reply:
What do you mean? This is just part of the bundle that will be deployed whenever we're using OLM 

This CRD is just copied from the autogenerated one.

Reply:
so the only way you'd see it at the EOD is with a `kubectl get crd ...`

Reply:
Had `\n` in the descriptions. When I dumped:

```
$ kubectl describe crd bpfprograms.bpfd.io
Name:         bpfprograms.bpfd.io
Namespace:    
Labels:       <none>
Annotations:  controller-gen.kubebuilder.io/version: v0.11.3
:
                  Description:  Condition contains details for one aspect of the current state of this API Resource. --- This struct is intended for direct use as an array at the field path .status.conditions.  For example, 
 type FooStatus struct{ // Represents the observations of a foo's current state. // Known .status.conditions.type are: "Available", "Progressing", and "Degraded" // +patchMergeKey=type // +patchStrategy=merge // +listType=map // +listMapKey=type Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type" protobuf:"bytes,1,rep,name=conditions"` 
 // other fields }
:
```
The `\n` didn't make a difference.

Reply:
It is ugly let me try and fix this

Reply:
So actually I can't @Billy99 This is from https://github.com/kubernetes/apimachinery/blob/v0.26.2/pkg/apis/meta/v1/types.go#L1464 which is part of the core API

Reply:
Just ignore the comment then. Just thought it looked weird, but not a big deal.